Fitness/Controllers/FilterViewController.swift

```diff
@@ -746,3 +533,224 @@ class FilterViewController: UIViewController, UICollectionViewDelegateFlowLayout
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let cell = tableView.cellForRow(at: indexPath) as! DropdownViewCell
+        var shouldAppend: Bool
+        
+        if(cell.checkBoxColoring.backgroundColor == .fitnessYellow){
+            cell.checkBoxColoring.backgroundColor = .white
+            shouldAppend = false
+        }else{
+            cell.checkBoxColoring.backgroundColor = .fitnessYellow
+            shouldAppend = true
+        }
+        
+        if (tableView == classTypeDropdown){
+            if(shouldAppend){
+                selectedClasses.append(cell.id)
+            }else{
+                for i in 0..<selectedClasses.count{
+                    let id = selectedClasses[i]
+                    if(id == cell.id){
+                        selectedClasses.remove(at: i)
+                        return
+                    }
+                }
+            }
+        }else{
+            if(shouldAppend){
+                selectedInstructors.append(cell.id)
+            }else{
+                for i in 0..<selectedInstructors.count{
+                    let id = selectedInstructors[i]
+                    if(id == cell.id){
+                        selectedInstructors.remove(at: i)
+                        return
+                    }
+                }
+            }
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 32
+    }
+    
+    func tableView(_ tableView: UITableView, heightForHeaderInSection section: Int) -> CGFloat {
+        return 55
+    }
+    
+    func tableView(_ tableView: UITableView, heightForFooterInSection section: Int) -> CGFloat {
+        var height = 0
+        if tableView == classTypeDropdown {
+            switch classTypeDropdownData.dropStatus {
+            case .up:
+                height = 0
+            case .half:
+                height = 32
+            case .down:
+                height = 32
+            default:
+                height = 0
+            }
+        } else if tableView == instructorDropdown {
+            switch instructorDropdownData.dropStatus {
+            case .up:
+                height = 0
+            case .half:
+                height = 32
+            case .down:
+                height = 32
+            default:
+                height = 0
+            }
+        }
+        return CGFloat(height)
+    }
+    
+    func tableView(_ tableView: UITableView, viewForFooterInSection section: Int) -> UIView? {
+        let footer = tableView.dequeueReusableHeaderFooterView(withIdentifier: DropdownFooterView.identifier) as! DropdownFooterView
+        
+        if tableView == instructorDropdown {
+            let gesture = UITapGestureRecognizer(target: self, action: #selector(self.dropHideInstructors(sender:) ))
+            footer.addGestureRecognizer(gesture)
+            if (instructorDropdownData.dropStatus == .half) {
```

Comment:
Maybe swiftlint will fix this, but yeah I’m going to stop leaving comments for removing `()` but please do it everywhere it exists. Thanks!